include/grpcpp/impl/codegen/interceptor.h

```diff
@@ -156,6 +156,9 @@ class InterceptorBatchMethods {
   /// started from interceptors without infinite regress through the interceptor
   /// list.
   virtual std::unique_ptr<ChannelInterface> GetInterceptedChannel() = 0;
+
+  // On a hijacked RPC, an interceptor can decide to fail a RECV MESSAGE op.
```

Comment:
I get the comment but can you describe the purpose of this operation in the API spec here? Basically, seeking additional detail.

Reply:
Added more details.